sdk/data/azappconfig/response_types.go

```diff
@@ -78,3 +79,94 @@ type SetSettingResponse struct {
 	// SyncToken contains the value returned in the Sync-Token header.
 	SyncToken SyncToken
 }
+
+// ArchiveSnapshotResponse contains the response from the ArchiveSnapshot method.
+type ArchiveSnapshotResponse struct {
+	Snapshot
+
+	// Link contains the information returned from the Link header response.
+	Link *string
+
+	// SyncToken contains the information returned from the Sync-Token header response.
+	SyncToken SyncToken
+}
+
+// RestoreSnapshotResponse contains the response from the RestoreSnapshot method.
+type RestoreSnapshotResponse struct {
+	Snapshot
+
+	// Link contains the information returned from the Link header response.
+	Link *string
+
+	// SyncToken contains the information returned from the Sync-Token header response.
+	SyncToken SyncToken
+}
+
+// NewGetSnapshotsPagerResponse contains the response from the NewGetSnapshotsPager method.
+type NewGetSnapshotsPagerResponse struct {
```

Comment:
For page responses, the response envelope name follows a slightly different pattern from CRUD operations. See https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/data/azappconfig/client.go#L228 for an example.